cmdrs-guide.html

```diff
@@ -0,0 +1,140 @@
+            <a href="http://edcodex.info/">Elite:Dangerous Codex</a>. There are a vast number of tools that you might
+            discover but these should be enough to get you started.
+        </p>
+        <h3>What am I supposed to do with these tools?</h3>
+        <p>
+            Like all great questions, the answer is; it depends. There are broadly 2 types of tools:
+        </p>
+        <ol>
+            <li>
+                Client tools, running on your PC.
+            </li>
+            <li>
+                Browser based tools, running on some remote server and accessed using your web browser.
+            </li>
+        </ol>
+        <p>
+            Client tools would need to be downloaded and run from your system. Most of the tools need to be run on the
+            same system that Elite Dangerous is running on although some tools might not need so.
+        </p>
+        <p>
+            The available tools can be used for anything from tracking and logging your activities, keeping track of
+            your resources and credits, to giving advice for how to build that perfect ship you are looking for.
+        </p>
+        <h3>How do these tools get their data?</h3>
+        <p>
+            There are 3 ways tools get their data:
+        </p>
+        <ol>
+            <li>
+                From <a href="#journal">journal files</a> generated by Elite Dangerous. Some use the data of these files
+                sent through EDDN.
+            </li>
+            <li>
+                From <a href="#capi">CAPI</a> once you authenticate your Frontier account with the tools.
+            </li>
+            <li>
+                From manual user input
+            </li>
+        </ol>
+        <h3>What are journal files, EDDN and CAPI?</h3>
+        <h4 id="journal">Journal Files</h4>
+        <p>
+            When you start Elite Dangerous and do activities in it, the game writes some particular files in the system
+            you are playing on. These files keep track of your activities by writing
+            <a href="https://www.w3schools.com/whatis/whatis_json.asp">JSON</a> data. These files are called the Journal
+            files. By default you can find these files at
+            <code>C:\Users\%userprofile%\Saved Games\Frontier Developments\Elite Dangerous</code> (Please note that
+            depending on your setup, the location might be different).
+        </p>
+        <p>
+            The files in this folder can track back years if you haven't changed your system. That means these files do
+            not carry over when you switch PCs. One way to carry them over is to manually copy from the old system and
+            paste them in the new system.
+        </p>
+        <h4 id="eddn">Elite Dangerous Data Network (EDDN)</h4>
+        <p>
+            Elite Dangerous Data Network (EDDN) is a community run message broker used by browser based tools. Client
```

Comment:
Reason I suggested to remove abbreviation in the header and add it to the body is just to align with what I would expect from docs. But I wouldn't have it listed twice if it is going to be used

If you want abbreviation in the header it would be better to have the paragraph starts with `EDDN is...` since the abbreviation will be used the most throughout the page

Reply:
I agree that might work. But I wanted to make this doc idiot proof and hence explaining stuff more than once if necessary. Still if the community feels strongly about this, I will remove it.